authz-module/index.md

```diff
@@ -0,0 +1,204 @@
+---
+parent:
+  title: Understanding the Authz Module
+order: 0
+description: Use the Cosmos SDK authz module to grant authorizations from one account (the granter) to another account (the grantee).
+---
+
+# Authz Module
+
+The authz ("authorization") module enables a granter to grant authorizations to a grantee that allows the grantee to execute messages on behalf of the granter.
+
+## Use Authz to Grant Authorizations
+
+By implementing the [authz module](https://docs.cosmos.network/v0.43/modules/authz/), Cosmos SDK application developers give users the ability to grant certain privileges to other users. For example, a user might want another user to vote on their behalf and so, rather than giving the other user access to their account, the user would grant an authorization that allows the other user to execute `MsgVote` on their behalf.
+
+How users decide to use the authz module is up to them. In one case, a validator might want to create a separate account for voting in order to keep their validator key more secure. In another case, a DAO might want to distribute authorizations to members of the DAO in which case a multisg account would grant authorizations to individual accounts and members of the DAO would be able to execute messages without requiring signatures from all the other members.
+
+In this tutorial, you spin up a single node network using the simulation application in Cosmos SDK (`simapp`), grant an authorization to another account, and then execute a message on behalf of the granter as the grantee.
+
+## Requirements
+
+Before you start the tutorial, you need to install the `simd` binary.
+
+Clone the `cosmos-sdk` repository:
+```
+git clone https://github.com/cosmos/cosmos-sdk
+```
+
+Change directories and check out `v0.43.0`:
+```
+cd cosmos-sdk && git checkout v0.43.0
+```
+
+Install the `simd` binary:
+```
+make install
+```
+
+Check to make sure the installation was successful:
+```
+simd version
+```
+
+You should see `0.43.0` printed to the console.
+
+## Configuration
+
+<!-- TODO: update tip to use `unsafe-reset-all` with better user experience -->
+
+::: tip
+If you have used `simd` before, you may already have a `.simapp` directory in you home directory. If you would like to keep the previous data, either save the directory to another location or use the `--home` flag and specify a different directory for each command in the following instructions. If not, remove the previous directory (`rm -rf ~/.simapp`).
```

Comment:
```suggestion
If you have used `simd` before, you might already have a `.simapp` directory in your home directory. To keep the previous data, either save the directory to another location or use the `--home` flag and specify a different directory for each command in the following instructions. If you don't want to keep the previous data, remove the previous directory (`rm -rf ~/.simapp`).
```

Reply:
I love words, I can talk about them all day long. Did you know that the Google developer documentation style guide is my best friend? (and is the resource that Tendermint docs rely on?)
I suggest that we use "might" to indicate possibility, see https://developers.google.com/style/word-list?hl=en#letter-m

Reply:
Thanks for the friendly reminder. I will re-review the style guide documentation.

Reply:
Thanks! I should have re-reviewed the style guide documentation.

Reply:
Thanks! I will try to make better use of the style guide.